packages/runtime/src/templates/getHandler.ts

```diff
@@ -57,10 +58,13 @@ const makeHandler = ({ conf, app, pageRoot, NextServer, staticManifest = [], mod
     require.resolve('./pages.js')
   } catch {}
 
+  const { appDir }: ExperimentalConfigWithLegacy = conf.experimental
   // Next 13.4 conditionally uses different React versions and we need to make sure we use the same one
-  overrideRequireHooks(conf)
+  // With the release of 13.5 experimental.appDir is no longer used.
+  // we will need to check if appDir exists to run requireHooks for older versions
+  if (appDir) overrideRequireHooks(conf.experimental)
   const NetlifyNextServer: NetlifyNextServerType = getNetlifyNextServer(NextServer)
-  applyRequireHooks()
+  if (appDir) applyRequireHooks()
```

Comment:
Do we want consistent brackets or are we happy with single-line `if` statements?

Reply:
Figured since the logic and return were pretty simple having it within a single line wouldn't cause confusion

Reply:
If we do we should probably update eslint, but IMO it's a lesser concern at the moment

Reply:
Yes - agreed.